runtime/doc/dev_theme.txt

```diff
@@ -67,8 +66,17 @@ Palettes
   (which is used for |hl-Normal|) as possible. They should have (reasonably)
   different hues to make them visually separable.
 
-- Each palette color should have a 256 colors variant with closest color
-  computed based on the perceptually uniform distance measure.
+- For 16 colors:
+
+    - Greys are not used and are replaced with foreground and background of
+      terminal emulator.
```

Comment:
```suggestion
      the terminal emulator's color scheme.
```

Reply:
I think "the foreground and background **colors** of the terminal emulator" is better